packages/office-ui-fabric-react/src/components/pickers/Suggestions/SuggestionsItem.styles.ts

```diff
@@ -67,8 +67,17 @@ export function getStyles(props: ISuggestionsItemStyleProps): ISuggestionsItemSt
       suggested && [
         classNames.isSuggested,
         {
-          background: semanticColors.menuItemBackgroundPressed,
           selectors: {
+            ':after': {
+              content: '""',
+              position: 'absolute',
+              left: 0,
+              top: 0,
+              bottom: 0,
+              right: 0,
+              border: `1px solid ${theme.palette.neutralSecondary}`,
```

Comment:
```suggestion
              border: `1px solid ${theme.semanticColors.focusBorder}`,
```

Use semanticColor from theme when possible (should have the same color as neutralSecondary)